whaledo/models/base.py

```diff
@@ -74,10 +71,10 @@ def predict(
             db_size -= 1
 
         k = min(k, db_size)
-        scores, topk_inds = probs.topk(dim=1, k=k, sorted=sorted)
+        scores, topk_inds = probs.topk(dim=1, k=k, sorted=sorted_)
         mask = self.threshold_scores(scores=scores)
         n_retrieved_per_query = mask.count_nonzero(dim=1)
-        mask_inds = mask.nonzero(as_tuple=True)
+        mask_inds = mask.nonzero(as_tuple=True)  # I get unexpected argument `as_tuple` here.
```

Comment:
It's very much a valid argument -- I use it extensively

Reply:
do you reckon it's a version issue? or just bad typing?

Reply:
`def nonzero(self): # real signature unknown; restored from __doc__`
ok - so it must be a generated type stub